File: Assignment 23/Example2.c
  /* 2. Accept character from user. if character is small display its corresponding capital character,and 
  if it small then display  its corresponding . in other cases display as it is  */

  #include<stdio.h>

  void Display(char ch )
  {
    if (ch >= 'a' && ch <= 'z')  
    {
        printf("Character is capital : %c\n", ch - 32);
    }
    else 
    {
        (ch >= 'A' && ch <= 'Z');
        printf("character is small : %c\n", ch + 32);
    }

  }
  int main()
  {
    char cValue=0;

    printf("Enter the character:\n");
    scanf("%c",&cValue);

    Display(cValue);
    
    return 0;
  }

File: Assignment 55/Example2.java
/*2. Write java program which acept array of chracters from user and replacce each capital character with its correspoing 
small character.

Input: b N e B R b A i G i
output:  4
*/

import java.util.Scanner;

class MyArray
{
    public int ArrayCountVowels(char Arr[])
    {
        int i = 0 ;
         int iCount = 0;

        for ( i = 0; i < Arr.length; i++) 
        {
            if (Arr[i] >= 'A' && Arr[i] <= 'Z') 
            {
                Arr[i] = (char)(Arr[i] + 32); 
                iCount++;
            }
        }
        return iCount;

    }
}
  
class Example2
{
    public static void main(String  A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter number of characters: ");
        int size = sobj.nextInt();
        char Arr[] = new char[size];

        System.out.println("Enter the characters:");
        for (int i = 0; i < size; i++) 
        {
            Arr[i] = sobj.next().charAt(0);
        }

        MyArray mobj = new MyArray();
        int iRet =  0 ;
         iRet = mobj.ArrayCountVowels(Arr); 

        System.out.println(iRet); 
        }
    }


